Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml

```diff
@@ -189,31 +189,31 @@
   - type: StorageFill
     contents:
     - id: FoodPizzaArnold
-      prob: 0.15
+      prob: 0.13
       orGroup: Pizza
     - id: FoodPizzaDank
-      prob: 0.15
+      prob: 0.05
       orGroup: Pizza
     - id: FoodPizzaSassysage
-      prob: 0.15
+      prob: 0.13
       orGroup: Pizza
     - id: FoodPizzaMargherita
-      prob: 0.15
+      prob: 0.13
       orGroup: Pizza
     - id: FoodPizzaMeat
-      prob: 0.15
+      prob: 0.13
       orGroup: Pizza
     - id: FoodPizzaMushroom
-      prob: 0.15
+      prob: 0.13
       orGroup: Pizza
     - id: FoodPizzaPineapple
-      prob: 0.15
+      prob: 0.12
       orGroup: Pizza
     - id: FoodPizzaVegetable
-      prob: 0.15
+      prob: 0.13
       orGroup: Pizza
     - id: FoodPizzaDonkpocket
-      prob: 0.15
+      prob: 0.05
       orGroup: Pizza
 
```

Comment:
This is pretty icky that maths is being done just to make the orGroup = 1?

Reply:
Ensured that it shouldn't need to equal 1, still made the slight changes to the pizza box because donk and dank pizza actually offer some slight benefits the others don't and should just be a little rarer.